docs/reference/argocd.md

```diff
@@ -44,7 +44,7 @@ Name | Default | Description
 [**StatusBadgeEnabled**](#status-badge-enabled) | `true` | Enable application status badge feature.
 [**TLS**](#tls-options) | [Object] | TLS configuration options.
 [**UsersAnonymousEnabled**](#users-anonymous-enabled) | `true` | Enable anonymous user access.
-[**Version**](#version) | v2.2.2 (SHA) | The tag to use with the container image for all Argo CD components.
```

Comment:
The version bump of the image is not part of this change. Has this been done before, and this is just docs update?

Reply:
Yes, We changed the container image while releasing 0.3.0.